packages/sources/src/api/handleError.js

```diff
@@ -0,0 +1,10 @@
+import { getSourcesApi } from './index';
+
+export const handleError = (id, { config: { url }, response: { data: { errors: [{ detail }] } } }) => getSourcesApi()
+.deleteSource(id)
+.catch((errorDelete) => {
+    throw `${url}: ${detail}. The source was not removed, try remove it later: ${errorDelete.response.data.errors[0].detail}`; }
+)
+.then(() => {
+    throw `${url}: ${detail}`;
```

Comment:
I suppose you want to throw multiple errors?

Reply:
`Promise.All `returns only the first error, so no, I just want to show the first returned error.

Reply:
Yeah i know i am just curious why there is throw in `then` branch.

Reply:
This is like an error 'post-process', the error is then catched by the outer catch branch.

Reply:
You can do mutations via graphql? I though its not supported.

Reply:
I don't understand what you mean